src/flux/task-plan.coffee

```diff
@@ -459,6 +463,8 @@ TaskPlanConfig =
 
     isStatsFailed: (id) -> !! @_stats[id]
 
+    hasChanged: (id) -> not _.isEqual(@exports.getChanged.call(@, id), @_local[id].defaultPlan)
```

Comment:
looks like `confirm-leave-mixin.cjsx`, `builder.cjsx`, and `plan-mixin.cjsx` all use `.isChanged`. If `.hasChanged` contains different logic than `.isChanged` then will the confirm dialog show up at different times?